app/logging.conf

```diff
@@ -0,0 +1,36 @@
+[loggers]
+keys=root,uicheckapp
```

Comment:
This seems incredibly long?

Are you pulling this from somewhere? This shouldn't be this long.

Reply:
In addition it includes `uicheckapp` as a key which we don't have?

Reply:
https://philstories.medium.com/fastapi-logging-f6237b84ea64 ?